package-lock.json

```diff
@@ -13484,9 +13484,9 @@
       }
     },
     "icu4c-data": {
-      "version": "0.63.2",
-      "resolved": "https://registry.npmjs.org/icu4c-data/-/icu4c-data-0.63.2.tgz",
-      "integrity": "sha512-vT6/47CcBzDemlhRzkL7dZLoNvuUWjjiuKZHMt5T4dbkKAqLBh7ZQ33GU13ecC/aIcMlIdBOqtF4uIYTgWML4Q==",
+      "version": "0.64.2",
```

Comment:
Can we revert these changes and just keep the changes to `mattermost-redux`?

Reply:
Sure. I did let it because it was automatically added by npm while installing the new mattermost-redux version but sure I'll remove it !